file.c

```diff
@@ -6537,6 +6537,142 @@ const char ruby_null_device[] =
  *  read-only, which is reported as <code>0444</code>.
  *
  *  Various constants for the methods in File can be found in File::Constants.
+ *
+ *  == What's Here
+ *
+ *  First, what's elsewhere. \Class \File:
+ *
+ *  - Inherits from class IO -- in particular, methods for creating,
+ *    reading, and writing files.
+ *  - Includes module FileTest.
+ *
+ *  \Class \File provides methods for:
+ *
+ *  === Creating
+ *
+ *  - ::new:: Creates a new file at the given file path; returns the file.
```

Comment:
`Opens the file at the given path`.  It doesn't create files at all by default, attempting to use `new` to create a new file raises an error unless you pass the appropriate mode as well.